### DISCUSSION TITLE
Dropdown - how do I control the styling of the chin space

### DISCUSSION THREAD

**Discussion by alan-copeland-keysight** [flet-dev/flet#2176] [Q&A]
### Question

Using flet 0.14.0, I'm trying to style the "chin space" of the Dropdown control (_change its background color for hover/selected_). I bet its not called the "chin space" but I don't know what else to call it. Its really the space that is:
1. below the Dropdown control text box, and 
2. surrounding the helper_text.

In this example, I've wrapped the Dropdown in a container with a beautiful pink border, and given the Dropdown.helper_text a nice yellow background, to show that extra "chin space". Any thoughts?

![image](https://github.com/flet-dev/flet/assets/69871845/e0a7c07f-2829-46c3-83ca-d762b825d7ba)

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
What do you mean by styling it?

**Reply by alan-copeland-keysight** [reply]
I really want the background to be transparent. Right now it has the styling of the drop-down box, so it looks like the drop-down overflows it's border.

**Reply by ndonkoHenri** [reply]
I am sorry but I don't know how it is done in Flutter.
If you find how it is done there, feel free to share it here and we will perhaps be able to implement it.

**Reply by alan-copeland-keysight** [reply]
Ok, I'll dig in a bit. Maybe I can find it's true name as well :)
